s3/replication/manager/src/s3replicationmanager/app.py

```diff
@@ -17,11 +17,176 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-from s3replicationcommon.log import Logger
+# import sys
+# import argparse
+# import json
+from aiohttp import web
+import logging
```

Comment:
may not be required as its imported indirectly at line 27 below

Reply:
done. logging module will not be imported from app.py anymore